### ISSUE TITLE
processBemJson: no tag property

### ISSUE THREAD

**Issue by zxqfox** [bem/bh-php#7]
Code:
```php
require "vendor/autoload.php";
$bh = new \BEM\BH();

$bh->match('button', function ($ctx) {
    $ctx->tag('button');
});

var_dump($bh->processBemJson([ 'block' => 'block' ]));
// [ 'block' => 'button', 'tag' => 'button' ]
```

Expected:
```
class BEM\Json#4 (9) {
  public $block =>
  string(5) "block"
  public $tag =>
  string(6) "button"
// ...
```

Actual: no tag property at all.

**Comment by zxqfox** [comment]
https://github.com/bem/bh/pull/96/files

**Comment by zxqfox** [comment]
Landed in 4.0.0
